src/Cache.php

```diff
@@ -37,12 +40,27 @@ final class Cache implements CacheInterface
      */
     private $handler;
 
+    /**
+     * @var string a string prefixed to every cache key so that it is unique globally in the whole cache storage.
+     * It is recommended that you set a unique cache key prefix for each application if the same cache
+     * storage is being used by different applications.
+     */
+    private $keyPrefix = '';
+
+    /**
+     * @var SerializerInterface the serializer to be used for serializing and unserializing of the cached data.
+     */
+    private $serializer;
+
+    private $keyNormalization = true;
+
     /**
      * @param \Psr\SimpleCache\CacheInterface cache handler.
      */
     public function __construct(\Psr\SimpleCache\CacheInterface $handler = null)
     {
         $this->handler = $handler;
+        $this->initSerializer();
```

Comment:
It's one liner that is clear about what it does. There is no need for separate method.